libs/designsystem/header/src/header.component.html

```diff
@@ -1,22 +1,19 @@
-
-<div class="container" [ngClass]="{ centered: centered }">
-  <div *ngIf="!!avatar" class="avatar">
-    <ng-content select="kirby-avatar"></ng-content>
-  </div>
-  <div *ngIf="!!flag" class="flag">
-    <ng-content select="kirby-flag"></ng-content>
-  </div>
-  <h1 *ngIf="!!title && !!value" class="key kirby-text-medium">
-    {{ title }}
-  </h1>
-  <h1 *ngIf="!!title && !value" class="title" [kirbyFitHeading]="fitHeadingConfig">
-    {{ title }}
-  </h1>
-  <h3 *ngIf="!!value" class="value kirby-text-display-3">{{ value }}<span class="value-unit kirby-text-normal semi-dark-text" *ngIf="!!valueUnit">{{ valueUnit }}</span></h3>
-  <div *ngIf="!!subtitle1" class="subtitle1 truncate kirby-text-small">
-    {{ subtitle1 }}
-  </div>
-  <div *ngIf="!!subtitle2" class="subtitle2 truncate kirby-text-small">
-    {{ subtitle2 }}
-  </div>
+<div *ngIf="!!avatar" class="avatar">
+  <ng-content select="kirby-avatar"></ng-content>
+</div>
+<div *ngIf="!!flag" class="flag">
+  <ng-content select="kirby-flag"></ng-content>
+</div>
+<h1 *ngIf="!!title && !!value" class="title kirby-text-medium">
+  {{ title }}
+</h1>
+<h1 *ngIf="!!title && !value" class="title" [kirbyFitHeading]="fitHeadingConfig">
+  {{ title }}
+</h1>
```

Comment:
Logically it would make sense to flip the order of these two headings as the small h1 (kirby-text-medium) should be shown when the h3 (value) is also shown. Maybe even consider grouping the small h1+h3 under the same *ngIf...